### ISSUE TITLE
"Nel json fornito..."?

### ISSUE THREAD

**Issue by mauromol** [italia/spid-sp-access-button#19]
Le linee guida sulle interfacce SPID pubblicate all'indirizzo:
https://www.spid.gov.it/assets/res/agid-spid-lg-interfacce-informazioni-idp-sp.pdf
che rimandano a questo repository GitHub per il pulsante "Entra con SPID" da usare negli SP, alla fine del paragrafo corrispondente dicono:

> Nel json fornito sono riportate tutte le diciture standard, avvisi e messaggi di errore.

Ebbene... dove sarebbe questo JSON e come si inserirebbe nel contesto del pulsante "Entra con SPID"?
Si tratta di un semplice refuso?

**Comment by mauromol** [comment]
Mi accorgo adesso che nel medesimo documento si fa riferimento ad un altro link per quanto riguarda questo JSON:

https://github.com/italia/spid-i18n

Link che però non funziona (404).

**Comment by barmassimo** [comment]
Con più di un anno di ritardo, mi accodo a questa richiesta: il link in oggetto è ancora errato.

**Comment by peppelinux** [comment]
Cari se potete e quando potete fare una PR per questo sarà mia cura unirla in master branch asap

**Comment by mauromol** [comment]
Il fatto è che secondo me questo JSON non esiste e non saprei neppure come inquadrarlo nell'ambito dell'access button (leggi: a cosa serve?).

**Comment by barmassimo** [comment]
Infatti, per ora ho copiato i testi presenti nelle immagini (solo italiano), e speriamo che basti.

Grazie, arrivederci

Massimo Barbieri
